30_ComputeShaderPathTracer/main.cpp

```diff
@@ -929,31 +905,26 @@ class ComputeShaderPathtracer final : public examples::SimpleWindowedApplication
 				// disregard surface/swapchain transformation for now
 				const auto viewProjectionMatrix = m_camera.getConcatenatedMatrix();
 
+				// compute view matrix
+				auto mv = viewMatrix;
+				auto mvp = viewProjectionMatrix;
+				core::matrix3x4SIMD normalMat;
+				mv.getSub3x3InverseTranspose(normalMat);
+				SBasicViewParametersAligned pc;
+
+				SBasicViewParametersAligned viewParams;
+				memcpy(viewParams.params.MV, mv.pointer(), sizeof(mv));
+				memcpy(viewParams.params.MVP, mvp.pointer(), sizeof(mvp));
+				memcpy(viewParams.params.NormalMat, normalMat.pointer(), sizeof(normalMat));
+
+				memcpy(pc.params.MV, mv.pointer(), sizeof(mv));
+				memcpy(pc.params.MVP, mvp.pointer(), sizeof(mvp));
+				memcpy(pc.params.NormalMat, normalMat.pointer(), sizeof(normalMat));
```

Comment:
only these two matrices need to be sent over

Reply:
Done